lib/services/github.ts

```diff
@@ -185,19 +185,16 @@ export class GithubService extends WorkerClient<string> implements ServiceListen
 				return false;
 			}
 
-			// Standard Express installation.
-			const app = express();
-			app.use(bodyParser.urlencoded({ extended: true }));
-			app.use(bodyParser.json());
+			const app = this.massageToExpress(listenerConstructor.server);
 
 			// Listen for Webhooks on the path specified by the client.
 			app.post(listenerConstructor.path, (req, res) => {
 				const eventType = req.get('x-github-event') || '';
 				const githubSignature = req.get('x-hub-signature') || '';
-				const payload = req.body;
+				const payload = JSON.parse(req.body);
```

Comment:
So, this is because now we have SyncBot which wants to be able to pass any content type. Why doesn't the express instance passed by SB instead specifically add a body parser for each type that it's going to process? The Github Service is only guaranteeing to process JSON, and nothing else (as nothing else is valid), so I really don't like the idea of letting it try and parse *anything*. It's essentially passing `any` to the body parser.

Reply:
Issue created https://github.com/resin-io-modules/resin-procbots/issues/398